packages/dev/loaders/src/glTF/2.0/Extensions/KHR_materials_dispersion.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type { Nullable } from "core/types";
+import { PBRMaterial } from "core/Materials/PBR/pbrMaterial";
+import type { Material } from "core/Materials/material";
+import type { IMaterial } from "../glTFLoaderInterfaces";
+import type { IGLTFLoaderExtension } from "../glTFLoaderExtension";
+import { GLTFLoader } from "../glTFLoader";
+import type { IKHRMaterialsDispersion } from "babylonjs-gltf2interface";
+
+const NAME = "KHR_materials_dispersion";
+
+/**
+ * [Specification](TODO)
+ * @since 5.0.0
+ */
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export class KHR_materials_dispersion implements IGLTFLoaderExtension {
+    /**
+     * The name of this extension.
+     */
+    public readonly name = NAME;
+
+    /**
+     * Defines whether this extension is enabled.
+     */
+    public enabled: boolean;
+
+    /**
+     * Defines a number that determines the order the extensions are applied.
+     */
+    public order = 174;
+
+    private _loader: GLTFLoader;
+
+    /**
+     * @internal
+     */
+    constructor(loader: GLTFLoader) {
+        this._loader = loader;
+        this.enabled = this._loader.isExtensionUsed(NAME);
+        if (this.enabled) {
+            // We need to disable instance usage because the attenuation factor depends on the node scale of each individual mesh
+            this._loader._disableInstancedMesh++;
+        }

```

Comment:
Is this necessary? Seems like a copy/paste. If it is necessary, the comment should be updated.

Reply:
Removed